seqr/views/apis/data_manager_api.py

```diff
@@ -322,6 +327,145 @@ def _update_individuals_sv_qc(json_records, user):
     'kl_temp_manton_orphan-diseases_cmg-samples_exomes_v1', 'Interview Exomes', 'v02_loading_test_project',
 ]
 
+@data_manager_required
+def receive_rna_seq_table(request):
+    if len(request.FILES) != 1:
+        return create_json_response({'errors': [f'Received {len(request.FILES)} files instead of 1']}, status=400)
+
+    stream = next(iter(request.FILES.values()))
+    filename = stream._name
+    if not filename.endswith('.tsv.gz'):
+        return create_json_response({'errors': [f'Invalid file extension for {filename}: expected ".tsv.gz"']}, status=400)
+
+    # save gzipped data to temporary file
+    uploaded_file_id = f'tmp_-_{datetime.now().isoformat()}_-_{request.user}_-_{filename}'
+    serialized_file_path = _get_upload_file_path(uploaded_file_id)
+    with open(serialized_file_path, 'wb') as f:
+        f.write(stream.read())
+
+    return create_json_response({
+        'uploadedFileId': uploaded_file_id,
+        'info': [f'Loaded gzipped file {filename}'],
```

Comment:
Because its cleaner and better practice in python 3. We never used it before because  its completely incompatible with python 2, but now that we have fully switched over theres no reason to be backwards compatible. It's not a big enough deal to actually go back and change all our strings, but when we are adding entirely new functions going forward I think it would be nice to start switching over. If you really dislike having mixed style, though, I can change them all back